view/GameView/SettingsScreen.py

```diff
@@ -57,15 +59,16 @@ def __init__(self, window: pygame.display, controller: ControllerGameView, paren
 
     def draw(self) -> None:
         self.manager.update(1 / self.settings.frame_rate)
-        self._update_label()
+        if not self.__spectator:
+            self._update_label()
 
         self.window.blit(self.background, (0, 0))
         self.manager.draw_ui(self.window)
 
     def receive_event(self, event: pygame.event.Event) -> None:
         self.manager.process_events(event)
 
-        if event.type == pygame.USEREVENT and event.user_type == pygame_gui.UI_BUTTON_PRESSED:
+        if event.type == pygame.USEREVENT and event.user_type == pygame_gui.UI_BUTTON_PRESSED and not self.__spectator:
```

Comment:
only have pause and shorcuts here and remove `and self.___spectator` from if below